docs/examples/pyang/index.md

```diff
@@ -0,0 +1,430 @@
+---
+layout: default
+title: "pyang examples"
+date: 2021-07-16 12:17:00 --0600
+categories:
+---
+
+## About Pyang
+
+pyang is a python program.
+
+We can use it to:
+
+- Validate YANG modules against YANG RFCs
+- Convert YANG modules into equivalent YIN module (XML)
+- Generate a tree representation of YANG models for quick visualization
+
+## Install Pyang
+
+```shell
+pip install pyang
+pip3 freeze | grep pyang
+```
+
+```shell
+pip3 freeze | grep pyang
+```
+
+## we need yang modules
+
+### Create a directory
+
+```shell
+mkdir yang_modules
+```
+
+### Clone the openconfig repository
```

Comment:
For consistency use OpenConfig instead of openconfig

Reply:
done, thanks